code/modules/mob/living/basic/space_fauna/bear/bear.dm

```diff
@@ -179,11 +182,16 @@
 		to_chat(src, span_notice("Your name is now <b>[new_name]</b>!"))
 		name = new_name
 
-/mob/living/simple_animal/hostile/bear/butter/AttackingTarget() //Makes the butter bear's attacks against vertical targets slip said targets
+/mob/living/basic/bear/butter/UnarmedAttack(atom/target, proximity_flag, list/modifiers) //Makes the butter bear's attacks against vertical targets slip said targets
 	. = ..()
-	if(isliving(target)) //we don't check for . here, since attack_animal() (and thus AttackingTarget()) will return false if your damage dealt is 0
-		var/mob/living/L = target
-		if((L.body_position == STANDING_UP))
-			L.Knockdown(20)
-			playsound(loc, 'sound/misc/slip.ogg', 15)
-			L.visible_message(span_danger("[L] slips on [src]'s butter!"))
+	if(!.)
+		return
+	if(!isliving(target))
+		return
+	var/mob/living/victim = target
+	if((!victim.body_position == STANDING_UP))
```

Comment:
buddy this is way too ambiguous, c'mon. if you want to nullcheck victim, we already did that in the `isliving()` check above there. do it like this

```suggestion
	if(victim.body_position != STANDING_UP)
```

completely confusing otherwise

Reply:
xd i forgeted